README.md

```diff
@@ -45,14 +63,24 @@ Run tests
 
 Generate documentation
 ----------------------
+Generate HTML documentation for Caffeine using [ford]:
 ```
 ford doc-generator.md
 ```
 Open `doc/html/index.htmtl` in a web browser.
```

Comment:
`index.htmtl` is probably a typo

Reply:
Good catch!  Thanks, @bonachea.  I'll push a fix momentarily. BTW, I finally bumped into the problem you predicted with the way I was setting `dt_sz`.  Commit 24a6b31252a0073b46c8ded810b54c8d83ad77b1 fixes it.